pkgs/applications/audio/stone-phaser/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, xorg, cairo, lv2, libjack2, mesa, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "stone-phaser";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "jpcima";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "180b32z8h9zi8p0q55r1dzxfckamnngm52zjypjjvvy7qdj3mfcd";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [
+    xorg.libX11 cairo lv2 libjack2 mesa
+  ];
+
+  patchPhase = ''
+    patch -d dpf -p 1 -i "`pwd`/resources/patch/DPF-bypass.patch"
```

```suggestion
    patch -d dpf -p 1 -i "resources/patch/DPF-bypass.patch"
```

Is `\`pwd\`` needed here? The unpack phase should have copied the src to the current directory I think.

---

Seems like it is, without I get: ``patch: **** Can't open patch file resources/patch/DPF-bypass.patch : No such file or directory``.

---

Oh interesting, my bad :+1: